WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/NewBlogDetailHeaderView.swift

```diff
@@ -0,0 +1,175 @@
+class NewBlogDetailHeaderView: UIView, BlogDetailHeader {
```

Comment:
Nit: are we going to rename this to `BlogDetailHeaderView` once it's complete?

Reply:
That's correct.